src/sentry/apidocs/examples/organization_examples.py

```diff
@@ -68,6 +68,50 @@ class OrganizationExamples:
         ),
     ]
 
+    RETRIEVE_SUMMARY_EVENT_COUNT = [
+        OpenApiExample(
+            "Successful response",
+            value={
+                "start": "2023-09-19T13:00:00Z",
+                "end": "2023-09-19T12:28:00Z",
+                "projects": [
+                    {
+                        "id": "1",
+                        "slug": "slug",
```

Comment:
These are visible to public so it's better to use more real life examples like "android-project" or something